json/deno.json

```diff
@@ -4,8 +4,8 @@
   "exports": {
     ".": "./mod.ts",
     "./types": "./types.ts",
-    "./concatenated-json-parse-stream": "./concatenated_json_parse_stream.ts",
```

Comment:
Should we do this as well?

Reply:
Yes, I think so.

Reply:
If we do this rename, I feel the rule becomes a bit obscure.

The rule here might be something like: `if the package name is included anywhere in module export name, then that part is removed from the export name`.

In that case the below items also includes package names:
- `@std/semver/is-semver`
- `@std/async/mux-async-iterator`

But I think we probably don't want to rename the above items.

Reply:
I'm mostly impartial, as I don't think such small details make much of a difference, either way. As long as the module name achieves its objective of signaling its purpose through its name, I think it's good. Happy to defer to you or hear other opinions.

Reply:
From some internal chats and a [small Discord survey](https://discord.com/channels/684898665143206084/775393009981849600/1257227741879865406), it seems people would rather keep `@std/json/concatenated-json-parse-stream`, so I reverted that change.